docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -179,14 +179,69 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Composants" href="/fr/guides/components/" />
 </CardGrid>
 
+### Asides
+
+Les encarts (également connues sous le nom de "aside" ou encore "élément en marge") sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
+
+Un `<Aside>` peut avoir un `type` optionnel de `note` (par défaut), `tip`, `caution` ou `danger`. La définition d'un attribut `title` remplace le titre par défaut de l'encart.
```

Comment:
```suggestion
Le composant `<Aside>` peut avoir un `type` optionnel de `note` (par défaut), `tip`, `caution` ou `danger`. La définition d'un attribut `title` remplace le titre par défaut de l'encart.
```

As "aside" does not mean anything in French, we cannot really use the same shortcut used in English.